README.md

```diff
@@ -1,2 +1,27 @@
-# keysets
-Custom keyboard shortcut presets for Warp, a blazingly fast modern terminal
+# Warp Keyboard Shortcut Presets
+
+This is an open source repository with custom keyboard shortcut presets for [Warp](https://www.warp.dev). We welcome and appreciate any contributions!
+
+### Disclaimer
+
+These presets are a work in progress and may be missing some functionality. As we continue to improve Warp and support more behaviors, they will be updated. If you have any questions or run into any issues, please reach out to us either on [Discord](https://discord.gg/warpdotdev) or in our [Issues Repository](https://github.com/warpdotdev/Warp)
+
+## Installation
+
+To use any of these keyboard presets, follow these steps:
+
+1. Confirm that you have a `~/.warp` directory. If you don't, create it with `mkdir ~/.warp`
```

Comment:
unrelated to this PR, but this would be great as a workflow @ianhodge @alokedesai